CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic
+Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
ironically there is a v to this version ^^

Reply:
Well spotted! I've seen this too but I kept it secret...